etc/pip_install/toree/toreeapp.py

```diff
@@ -91,6 +91,7 @@ def create_kernel_json(self, location, interpreter):
         try:
             py4j_zip = list(filter( lambda filename: "py4j" in filename, python_lib_contents))[0]
         except:
+            py4j_zip = None
```

Comment:
So rather than just setting this to `None` here, can you change the kernel_spec.env lower down to not set PYTHON_PATH, since it will be incorrect in this case.